src/presentation/cli/middleware/checkEnvs.go

```diff
@@ -12,12 +12,14 @@ import (
 )
 
 var requiredEnvVars = []string{
-	"PRIMARY_VHOST",
+	"ACCOUNT_API_KEY_SECRET",
 	"JWT_SECRET",
+	"PRIMARY_VHOST",
 	"UAK_SECRET",
 }
 
 var envVarsToGenerateIfEmpty = []string{
+	"ACCOUNT_API_KEY_SECRET",
 	"JWT_SECRET",
 	"UAK_SECRET",
```

Comment:
I think UAK was User Api Key. Check the code, you may be able to remove it, not sure.

Reply:
To be honest, it's not even being used. I just removed it.